build.gradle.kts

```diff
@@ -0,0 +1,290 @@
+import com.github.vlsi.gradle.crlf.CrLfSpec
+import com.github.vlsi.gradle.crlf.LineEndings
+import com.github.vlsi.gradle.properties.dsl.props
+
+plugins {
+    `java-library`
+    id("com.github.johnrengelman.shadow")
+    id("com.github.vlsi.crlf")
+    id("com.github.vlsi.gradle-extensions")
+    id("com.github.vlsi.stage-vote-release")
+}
+
+val skipJavadoc by props()
+val enableMavenLocal by props()
+val enableGradleMetadata by props()
+
+val String.v: String get() = rootProject.extra["$this.version"] as String
+
+val buildVersion = "darklaf".v + releaseParams.snapshotSuffix
+println("Building Darklaf $buildVersion")
+println("             OS: " + System.getProperty("os.name"))
```

Comment:
I'm inclined to remove this as the user often knows which OS they are using.

Reply:
Can be removed, no problem :)